extensions/mentions/src/ConfigureMentions.php

```diff
@@ -144,24 +144,24 @@ private function configureGroupMentions(Configurator $config)
         $tagName = 'GROUPMENTION';
 
         $tag = $config->tags->add($tagName);
-        $tag->attributes->add('displayname');
+        $tag->attributes->add('groupname');
         $tag->attributes->add('icon');
         $tag->attributes->add('color');
         $tag->attributes->add('id')->filterChain->append('#uint');
 
         $tag->template = '
             <xsl:choose>
                 <xsl:when test="@deleted != 1">
-                    <span class="GroupMention" style="background: {@color}">@<xsl:value-of select="@displayname"/><i class="icon {@icon}"></i></span>
+                    <span class="GroupMention" style="background: {@color}">@<xsl:value-of select="@groupname"/><i class="icon {@icon}"></i></span>
                 </xsl:when>
                 <xsl:otherwise>
-                    <span class="GroupMention GroupMention--deleted">@<xsl:value-of select="@displayname"/></span>
+                    <span class="GroupMention GroupMention--deleted" style="background: {@color}">@<xsl:value-of select="@groupname"/><i class="icon {@icon}"></i></span>
```

Comment:
is color available when the group is deleted?

Reply:
This is needed for the _preview_ post:

With:
![image](https://user-images.githubusercontent.com/16573496/197490661-a36644af-3281-4467-8c29-d48eb559dce8.png)


Without:
![image](https://user-images.githubusercontent.com/16573496/197490794-ffafe00d-f373-4214-bf12-b58e190a81f4.png)